File: read.cc

#include <iostream>
#include <string>
#include <fstream>

using namespace std;

int main(){
    string s; 
    ifstream f;

    f.open("input.txt");
    while(getline(f, s)){
        cout << s << endl;       
    }
    f.close(); 

    return 0;
}

File: read.py
#!/usr/bin/env python3
import sys
fname='input.txt'
try:
    f = open(fname, 'r')
except OSError as e:
    print(f"OS error ({e.errno}): '{e.strerror}' open file: {fname} ")
    sys.exit()

with f:
    try:
        print(f.read())
    except IOError as e:
        print("Error ({0}): '{1}' while reading file: {2}".format(e.errno, e.strerror,fname))
    finally:
        f.close()
